device/daita.go

```diff
@@ -0,0 +1,301 @@
+//go:build daita
+// +build daita
+
+package device
+
+import (
+	"time"
+	"unsafe"
+)
+
+// #include <stdio.h>
+// #include <stdlib.h>
+// #include "../cmaybenot/cmaybenot.h"
+// #cgo LDFLAGS: -L${SRCDIR}/../cmaybenot/target/release -l:libcmaybenot.a -lm
+import "C"
+
+type MaybenotDaita struct {
```

Comment:
Not sure I agree, the struct is extremely maybenot-specific, you need to pass maybenot arguments when initializing it, etc. Also, we have the `Daita` interface type (which is not as maybenot-specific) that represents the external interface. Once we move `MaybenotDaita` to its own module, this will make more sense?

Reply:
Is it possible to name the struct more closely to the corresponding one in the Maybenot crate?

By convention, interface names always end in `er` and are typically not repeated in the names of implementing structs.